ui/src/tasks/components/TaskRunsPage.tsx

```diff
@@ -0,0 +1,65 @@
+// Libraries
+import React, {PureComponent} from 'react'
+
+// Types
+import {Run} from '@influxdata/influx'
+import {IndexList} from 'src/clockface'
+import {taskRuns} from '../dummyData'
+import {Page} from 'src/pageLayout'
+
+interface Props {
+  taskID: string
+  runs: Run[]
+}
+
+const dummyData = taskRuns
+
+class TaskRunsPage extends PureComponent<Props> {
+  public render() {
+    return (
+      <>
+        <Page titleTag="Runs">
+          <Page.Header fullWidth={false}>
+            <Page.Header.Left>
+              <Page.Title title="Runs" />
+            </Page.Header.Left>
+            <Page.Header.Right />
+          </Page.Header>
+          <Page.Contents fullWidth={false} scrollable={true}>
+            <div className="col-xs-12">
+              <IndexList>
+                <IndexList.Header>
+                  <IndexList.HeaderCell columnName="Requested At" width="10%" />
+                  <IndexList.HeaderCell columnName="Started At" width="10%" />
+                  <IndexList.HeaderCell columnName="Finished At" width="15%" />
+                  <IndexList.HeaderCell columnName="Status" width="10%" />
+                  <IndexList.HeaderCell columnName="Schedule For" width="10%" />
+                  <IndexList.HeaderCell columnName="" width="45%" />
+                </IndexList.Header>
+                <IndexList.Body emptyState={<></>} columnCount={6}>
+                  {this.listRuns}
+                </IndexList.Body>
+              </IndexList>
+            </div>
+          </Page.Contents>
+        </Page>
+      </>
+    )
+  }
+
+  public get listRuns(): JSX.Element[] {
+    const taskRuns = dummyData.map(t => (
+      <IndexList.Row key={`task-id--${t.id}`}>
+        <IndexList.Cell>{t.requestedAt.toDateString()}</IndexList.Cell>
+        <IndexList.Cell>{t.startedAt.toDateString()}</IndexList.Cell>
```

Comment:
We might want to be a bit more granular than day here, i.e. if I have multiple runs of a task in the same day.

Reply:
Fixed it! :)